src/components/draw/DrawDirective.js

```diff
@@ -110,6 +110,9 @@ goog.require('ga_permalink');
       var updateHelpTooltip = function(overlay, type, drawStarted, onFirstPoint,
           onLastPoint) {
         var helpMsg = $translate.instant('draw_start_' + type);
+        if (type == 'marker' || type == 'annotation') {
+          helpMsg = $translate.instant('draw_next_' + type);
```

Comment:
I know, but the translation is there for next and it makes sense.

Wasn't sure if I could just replace draw_next_marker with draw_start_marker in the translations.